src/test/java/seedu/guest/storage/JsonAdaptedGuestTest.java

```diff
@@ -37,103 +37,103 @@ public class JsonAdaptedGuestTest {
 
     @Test
     public void toModelType_validPersonDetails_returnsPerson() throws Exception {
```

Comment:
Should this be toModelType_validGuestDetails_returnsGuest()?

Reply:
fixed